loxone.go

```diff
@@ -99,6 +99,54 @@ type Control struct {
 	Cat        string
 	States     map[string]interface{} // Can be an array or a string
 	Details    ControlDetails
+	Statistic  ControlStatistic
+}
+
+// StatisticalNames returns a list of names for the given control and
+// the statistical data the control writes. first we will loop over the
+// outputs values within the statistics of the control, trying to find
+// a state with a matching uuid. If no entry was found, we will use the
+// name of the statistic entry in the output array.
+func (cntl *Control) StatisticalNames() (names []string) {
+	var name string
+	// loop over the statistic outputs
```

Comment:
name declaration can be done on the loop to avoid the reset at the end of the loop